### PR TITLE
Initial note list layout

### PR DESCRIPTION
**Pull request by bherbst**
Adds an initial team match note list with some static data

image

### CODE REVIEW

File: ui/src/routes/notes/[teamNumber]/+page.svelte [hunk 1]
```diff
@@ -1,7 +1,50 @@
...
+				note: "Hungry for carpet",
+				noteId: "1",
+				tournamentLevel: "Practice",
+				matchNumber: 1,
+				playNumber: 1,
+				teamNumber: parseInt(params.teamNumber!),
+				timeAdded: "Wed Aug 7 2024 00:00:00 GMT+0000 (Coordinated Universal Time)",
+				timeUpdated: "Wed Aug 7 2024 00:00:00 GMT+0000 (Coordinated Universal Time)",
+				isDeleted: false,
+				whoAdded: "", // TODO will this be a usable format?
```
Comment (333fred) [inline ui/src/routes/notes/[teamNumber]/+page.svelte]: We can look at updating the model, but we'll need to coordinate with @kerrydh if we do make any changes, as he'll likely need to update the legacy app.

**Review by 333fred — COMMENTED** [review]
Need to run prettier.

File: ui/src/routes/notes/[teamNumber]/+page.svelte (hunk 1 above)
Comment (bherbst) [reply ui/src/routes/notes/[teamNumber]/+page.svelte]: Not necessarily looking to update the model, I just want to get see what this looks like in practice. I'm _assuming_ that it'll be the FTA's user ID, just want to confirm.

Comment (333fred) [reply ui/src/routes/notes/[teamNumber]/+page.svelte]: Yes, it should be.

**Review by 333fred — APPROVED** [review]

**Pull request merged**